go1.x/hello-img/{{cookiecutter.project_name}}/hello-world/main.go

```diff
@@ -1,47 +1,24 @@
 package main
 
 import (
-	"errors"
 	"fmt"
-	"io"
-	"net/http"
 
 	"github.com/aws/aws-lambda-go/events"
 	"github.com/aws/aws-lambda-go/lambda"
 )
 
-var (
-	// DefaultHTTPGetAddress Default Address
-	DefaultHTTPGetAddress = "https://checkip.amazonaws.com"
-
-	// ErrNoIP No IP found in response
-	ErrNoIP = errors.New("No IP in HTTP response")
-
-	// ErrNon200Response non 200 status code in response
-	ErrNon200Response = errors.New("Non 200 Response found")
-)
-
 func handler(request events.APIGatewayProxyRequest) (events.APIGatewayProxyResponse, error) {
-	resp, err := http.Get(DefaultHTTPGetAddress)
-	if err != nil {
-		return events.APIGatewayProxyResponse{}, err
-	}
-
-	if resp.StatusCode != 200 {
-		return events.APIGatewayProxyResponse{}, ErrNon200Response
-	}
-
-	ip, err := io.ReadAll(resp.Body)
-	if err != nil {
-		return events.APIGatewayProxyResponse{}, err
-	}
+	var greeting string
+	sourceIP := request.RequestContext.Identity.SourceIP
```

Comment:
what kind of intermittent errors are you running into? Most of our other examples also use a similar pattern. Making a change in one should also mean we should standardize across all.

Reply:
Read https://github.com/aws/aws-sam-cli-app-templates/issues/421 for additional context. Though I agree in principle, sounds like this should break other init templates too. 🤔

Reply:
I typically just use the Go template so I can't speak to the behavior in others. Though for a minimal example, I do agree with @moelasmar's comment in #421 that the hello-world example is doing too much and particularly should not rely on an API call to an external service for local testing.